src/App.js

```diff
@@ -6,8 +6,8 @@ import SignInPage from "./pages/SignInPage";
 function App() {
   return (
     // <Homepage/>
-    //<SignUpPage />
```

Comment:
Please remove all commented out code from the PR. The main branch should not have commented out code.

Reply:
Addressed. The commented-out codes have now been removed. Thanks!